cpp_linter/run.py

```diff
@@ -61,10 +61,18 @@ def set_exit_code(override: Optional[int] = None) -> int:
         The exit code that was used. If the ``override`` parameter was not passed,
         then this value will describe (like a bool value) if any checks failed.
     """
-    exit_code = override if override is not None else bool(Globals.OUTPUT)
+    exit_code = (
+        override
+        if override is not None
+        else (Globals.format_failed_count + Globals.tidy_failed_count)
+    )
```

Comment:
Using `bool(Globals.OUTPUT)` always returns true since revision of comment contents (see changes in #32). So, this enhancement fixes that ambiguity while implementing the new outputs. 🚀